app/src/main/res/layout-ar-land-v17/audio_panel.xml

```diff
@@ -96,17 +96,30 @@
             android:text="@string/play_verses_range"
             android:textAppearance="@style/PanelLabel"
             />
-        <com.quran.labs.androidquran.view.QuranSpinner
-            android:id="@+id/repeat_range_spinner"
+        <com.shawnlin.numberpicker.NumberPicker
```

Comment:
what do you think about using he default Android `NumberPicker`? the only downside is we can't style it on api levels before 24, but i think that is ok?